### ISSUE TITLE
what is next 

### ISSUE THREAD

**Issue by nourelhoda89** [nourelhoda89/automobile#15]
@Calvarez14
please make sure your parts are done very soon, for engine you can simply extend engine and just change the toString method , same with wheels, extends wheels and change toString, I think it will work like that. after that we need to start creating trim builders.
@Catyell2 
you need to create each of your trim builders which extends DodgeRamBuilderInterface  that is already available for us

**Comment by anthonykulis** [comment]
also, now that you have issues with the flowcharts (which is fine i guess), make sure you tag that issue in your commit (e.g. `fixes #n`, where `n` is the issue number and if fixes *fixes* it) . For each issue you created, you need to have one PR to work on it. **Every time** you commit work to that, you will need to reference it. Why the PR is a better way to go imo. Once the PR is complete, make sure to put in the PR commit message `closes #n` so that upon merge it auto-closes it. If that is forgotten, just close it manually.

**Comment by nourelhoda89** [comment]
https://www.ramtruck.ca/en/1500/spec 

Hey guys, this is a helpful website for finding the engine details .
 I just finished coding the Limited builder,but took me quite some time, so please don't wait to start working on them till Monday night, I won't have time to review them! For tomorrow  I have few question for Mr. Kulis, then we can discuss the builders and how they work. Also, I had to change few things in wheel and engine and some other things to make the code work , so please update you repo! 
@Calvarez14  Cesar please don't forget to open a  new pull request  for each builder so I can track your work, even if your stuff are not ready to be merged.

**Comment by nourelhoda89** [comment]
I fixed all the issues we had this morning , I made a lot of changes  and committed them separately, you can view them under commit. also to use both the askForTruckDrive and askForCabAndBox methods I created TruckDodgeRamBuilderInterface   that extend DodgeRamBuilderInterface  and implemented  TDBI in the builder.

**Comment by anthonykulis** [comment]
For u, I guess that's it. Help your team. The help you given them are a
percentage of your grade

**Comment by nourelhoda89** [comment]
@Calvarez14 @Catyell2 as you work on your builders tag me in any issues you have, I am ready to help! I will also be reviewing your commits on the pull request!

**Comment by Catyell2** [comment]
Will do.
